src/components/card/VideoCard.jsx

```
@@ -1,15 +1,12 @@
 import React from "react";
+import { nameElipse } from "../../utils/common/nameElipse";
 
 const VideoCard = ({ video }) => {
```

ESLint

[error] 'video' is missing in props validation

(react/prop-types)